src/css/components/po-tooltip/po-tooltip.css

```
@@ -4,28 +4,32 @@
   }
 
   --shadow-tooltip-shadow-balloon: var(--shadow-tooltip);
-
-  --tooltip-border-radius: 3px;
 }
 
 .po-tooltip {
-  background-color: var(--color-tooltip-background-color-balloon);
-  border-radius: var(--tooltip-border-radius);
+  background-color: var(--color-tooltip-background-color-balloon, var(--color));
+  border-radius: var(--border-radius);
   box-shadow: var(--shadow-tooltip-shadow-balloon);
-  max-width: 246px;
-  padding: 8px 16px;
+  max-width: 15rem;
+  padding: var(--spacing-squish-md);
   position: fixed;
   z-index: 1000;
+  opacity: 0;
 }
 
 .po-tooltip-arrow {
-  background-color: var(--color-tooltip-background-color-balloon);
+  background-color: var(--color-tooltip-background-color-balloon, var(--color));
+  opacity: 0.9;
 }
 
 .po-tooltip-content {
   @apply --font-tooltip-font-text;
-  color: var(--color-tooltip-color-text-balloon);
+  color: var(--color-tooltip-color-text-balloon, var(--text-color));
   font-style: normal;
+  font-weight: var(--font-weight-normal);
+  font-size: var(--font-size-sm);
   white-space: normal;
   word-wrap: break-word;
+  text-align: left;
+  opacity: 0.9;
```

Ficou faltando o `text-align: left` conforma handoff.

```suggestion
  opacity: 0.9;
  text-align: left;
```

---

O text-align está na linha 33